src/main/java/org/onebusaway/alexa/constant/Prompt.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2016 Sean J. Barbeau (sjbarbeau@gmail.com),
```

Comment:
Also please change copyright date from `2016` to `2016-2019`  in all headers for modified files.

Reply:
Done